users/static/users/scripts/update_account_popup.js

```diff
@@ -0,0 +1,24 @@
+$(function () {
+    $("#dialog_user_account_update").dialog ({
+        modal: true,
+        autoOpen: false,
+        buttons : [
+          {
+            text: "No",
+            click: function () {
+              $(this).dialog('close');
+            }
+          },
+          {
+            text: "Yes",
+            click: function () {
+                $('form').submit()
+            }
+          }
+        ]
+    }).prev().find(".ui-dialog-titlebar-close").hide ();
+
+    $("#opener_user_account_update").click(function () {
+        $('#dialog_user_account_update').dialog('open');
+    });
+});
```

Comment:
There's a chance we might have to change Martyna's script as well, so they both turn into a single script. It's beyond necessity, so if you're busy with other tasks at hand you can skip it and an issue for this will be made.